workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/Model/Details/UnityEventDetails.cs

```diff
@@ -1,19 +1,21 @@
-using Improbable.CodeGeneration.Utils;
+using Improbable.Gdk.CodeGeneration.Model.SchemaBundleV1;
+using Improbable.Gdk.CodeGeneration.Utils;
 
 namespace Improbable.Gdk.CodeGenerator
 {
     public class UnityEventDetails
     {
-        public string EventName => Formatting.SnakeCaseToCapitalisedCamelCase(eventDefinition.Name);
-        public string CamelCaseEventName => Formatting.SnakeCaseToCamelCase(eventDefinition.Name);
-        public string FqnPayloadType => CommonDetailsUtils.GetCapitalisedFqnTypename(eventDefinition.RawType.TypeName);
-        public uint EventIndex => eventDefinition.EventIndex;
+        public string EventName { get; }
```

Comment:
do we need them to be properties? they could also just be fields with a readonly attribute

Reply:
They could be - the properties give a bit more flexibility imo